### ISSUE TITLE
r9796 wipes out weapons.ini

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#739]
```
What steps will reproduce the problem?
1. Start Hedgewars 0.9.19
2. Create some custom weapon schemes
3. Close Hedgewars.
4. Start Hedgewars r9796 on the same config dir
5. Check the list of available weapon schemes

What is the expected output? What do you see instead?
I expect the old weapon schemes to be kept.
Instead the weapons list is resetted to the default. All prior entries are 
wiped out without question.
If you compare the weapons.ini before step 4 and after step 5, you will see the 
same result.

Luckily, I had a backup of the weapon schemes at hand, so no drama for me.

Please fix this before release or at the very least warn players to back up 
their weapons.ini or their entire .hedgewars directory before updating to (the 
coming) 0.9.20.

What version of the product are you using? On what operating system?
Hedgewars 0.9.19 vs. Hedgewars r9796 on GNU/Linux.
```

Original issue reported on code.google.com by `almikes@aol.com` on 24 Dec 2013 at 1:02
* Merged into: #303

**Comment by GoogleCodeExporter** [comment]
Original comment by `kyberneticist@gmail.com` on 24 Dec 2013 at 2:28
* Changed state: **Duplicate**
